### DISCUSSION TITLE
[General]: Thank you

### DISCUSSION THREAD

**Discussion by silllli** [davidjerleke/embla-carousel#227] [General]
I just want to say thank you @davidjerleke —this library works really great and I like the “do one thing and do it well” approach. Everything feels very clean and simple, not as hacky as many other carousel libraries.

🍻🍻🍻

**Comment by davidjerleke** [comment]
Thank you @silllli,

Enjoy and cheers!
David

**Comment by andycamara** [comment]
I second that. Best carousel I've ever used. Seamless and no b.s 
Thanks dude - I'll be using this on all my projects moving forward 🤙🏼
